### ISSUE TITLE
WebSocketEndpointFactory failed to initialize within the Lucee engine (startup-hook).

### ISSUE THREAD

**Issue by webonix** [lucee/extension-websocket#1]
On Lucee 6, a restart is required to make websocketInfo() work
<img width="837" alt="image" src="https://github.com/lucee/extension-websocket/assets/217567/df5618cd-b068-412b-8de3-20c2ee808ce2">

**Comment by webonix** [comment]
related ?
```
2024-04-23 15:18:41 java.lang.NoSuchMethodException: lucee.runtime.engine.CFMLEngineImpl.getConfigServerImpl()
2024-04-23 15:18:41     at java.base/java.lang.Class.getDeclaredMethod(Class.java:2475)
2024-04-23 15:18:41     at lucee.loader.engine.CFMLEngineFactory.getConfigServer(CFMLEngineFactory.java:927)
2024-04-23 15:18:41     at lucee.loader.engine.CFMLEngineFactory._restart(CFMLEngineFactory.java:864)
2024-04-23 15:18:41     at lucee.loader.engine.CFMLEngineFactory.restart(CFMLEngineFactory.java:841)
2024-04-23 15:18:41     at lucee.runtime.config.ConfigAdmin.restart(ConfigAdmin.java:3777)
2024-04-23 15:18:41     at lucee.runtime.tag.Admin.doRestart(Admin.java:952)
2024-04-23 15:18:41     at lucee.runtime.tag.Admin._doStartTag(Admin.java:867)
2024-04-23 15:18:41     at lucee.runtime.tag.Admin.doStartTag(Admin.java:364)
2024-04-23 15:18:41     at restart_cfm$cf.call(/admin/restart.cfm:5)
2024-04-23 15:18:41     at lucee.runtime.PageContextImpl._doInclude(PageContextImpl.java:1029)
2024-04-23 15:18:41     at lucee.runtime.PageContextImpl._doInclude(PageContextImpl.java:952)
2024-04-23 15:18:41     at lucee.runtime.listener.ModernAppListener._onRequest(ModernAppListener.java:220)
2024-04-23 15:18:41     at lucee.runtime.listener.ModernAppListener.onRequest(ModernAppListener.java:107)
2024-04-23 15:18:41     at lucee.runtime.PageContextImpl.execute(PageContextImpl.java:2716)
2024-04-23 15:18:41     at lucee.runtime.PageContextImpl._execute(PageContextImpl.java:2702)
2024-04-23 15:18:41     at lucee.runtime.PageContextImpl.executeCFML(PageContextImpl.java:2673)
2024-04-23 15:18:41     at lucee.runtime.engine.Request.exe(Request.java:45)
2024-04-23 15:18:41     at lucee.runtime.engine.CFMLEngineImpl._service(CFMLEngineImpl.java:1259)
2024-04-23 15:18:41     at lucee.runtime.engine.CFMLEngineImpl.serviceCFML(CFMLEngineImpl.java:1205)
2024-04-23 15:18:41     at lucee.loader.engine.CFMLEngineWrapper.serviceCFML(CFMLEngineWrapper.java:97)
2024-04-23 15:18:41     at lucee.loader.servlet.CFMLServlet.service(CFMLServlet.java:51)
2024-04-23 15:18:41     at javax.servlet.http.HttpServlet.service(HttpServlet.java:741)
2024-04-23 15:18:41     at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
2024-04-23 15:18:41     at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
2024-04-23 15:18:41     at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
2024-04-23 15:18:41     at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
2024-04-23 15:18:41     at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
2024-04-23 15:18:41     at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:199)
2024-04-23 15:18:41     at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96)
2024-04-23 15:18:41     at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:490)
2024-04-23 15:18:41     at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
2024-04-23 15:18:41     at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
2024-04-23 15:18:41     at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:668)
2024-04-23 15:18:41     at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
2024-04-23 15:18:41     at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
2024-04-23 15:18:41     at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:408)
2024-04-23 15:18:41     at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:66)
2024-04-23 15:18:41     at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:770)
2024-04-23 15:18:41     at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1415)
2024-04-23 15:18:41     at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
2024-04-23 15:18:41     at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
2024-04-23 15:18:41     at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
2024-04-23 15:18:41     at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
2024-04-23 15:18:41     at java.base/java.lang.Thread.run(Thread.java:829)
```

**Comment by michaeloffner** [comment]
this is the origin of the issue 
The OSGi Bundle with name [org.lucee.websocket.extension] in version [0.0.0.62-ALPHA] is not available locally [ (/Users/mic/Test/test-cfconfig/lucee-server/bundles)] or from the update provider [ (https://update.lucee.org)].;org.osgi.framework.BundleException: The OSGi Bundle with name [org.lucee.websocket.extension] in version [0.0.0.62-ALPHA] is not available locally [ (/Users/mic/Test/test-cfconfig/lucee-server/bundles)] or from the update provider [ (https://update.lucee.org)].
	at lucee.runtime.osgi.OSGiUtil._loadBundle(OSGiUtil.java:825)
	at lucee.runtime.osgi.OSGiUtil.loadBundle(OSGiUtil.java:680)
	at lucee.runtime.osgi.OSGiUtil$BundleDefinition.getBundle(OSGiUtil.java:2083)
	at lucee.commons.lang.ClassUtil.loadClassByBundle(ClassUtil.java:179)
	at lucee.commons.lang.ClassUtil.loadClassByBundle(ClassUtil.java:163)
	at lucee.transformer.library.ClassDefinitionImpl.getClazz(ClassDefinitionImpl.java:120)
	at lucee.transformer.library.ClassDefinitionImpl.getClazz(ClassDefinitionImpl.java:111)
	at lucee.runtime.config.ConfigWebFactory._loadStartupHook(ConfigWebFactory.java:4297)
	at lucee.runtime.config.ConfigWebFactory.load(ConfigWebFactory.java:682)
	at lucee.runtime.config.ConfigServerFactory.load(ConfigServerFactory.java:257)
	at lucee.runtime.config.ConfigServerFactory.reloadInstance(ConfigServerFactory.java:236)
	at lucee.runtime.config.ConfigAdmin._reload(ConfigAdmin.java:340)
	at lucee.runtime.config.ConfigAdmin._storeAndReload(ConfigAdmin.java:318)
	at lucee.runtime.config.ConfigAdmin.updateRHExtension(ConfigAdmin.java:4999)
	at lucee.runtime.config.ConfigAdmin.updateRHExtension(ConfigAdmin.java:4567)
	at lucee.runtime.config.ConfigAdmin._updateRHExtension(ConfigAdmin.java:4547)
	at lucee.runtime.config.DeployHandler.deploy(DeployHandler.java:98)
	at lucee.runtime.engine.Controler.control(Controler.java:236)
	at lucee.runtime.engine.Controler$ControlerThread.run(Controler.java:118)

**Comment by michaeloffner** [comment]
fixed in Lucee 6.1.0.134

**Comment by webonix** [comment]
@michaeloffner @micstriit @zspitzer 

where can I download `Lucee 6.1.0.134`

I am only seeing .100 BETA on downloads page
and with that version I am not able to access Lucee Admin
